internal/stack/shellinit.go

```diff
@@ -131,6 +132,14 @@ func getShellName(exe string) string {
 func detectShell() (string, error) {
 	ppid := os.Getppid()
 	parentInfo, err := getParentInfo(ppid)
+	if errors.Is(err, types.ErrNotImplemented) {
+		// Sysinfo doesn't implement some features in some platforms.
+		// This mainly affects osx when building without CGO.
+		// Assume bash in that case.
+		// See https://github.com/elastic/elastic-package/issues/1030.
+		logger.Debugf("Failed to determine parent process info while detecting shell, will assume bash")
```

Comment:
nit. No variable used in the message.
```suggestion
		logger.Debug("Failed to determine parent process info while detecting shell, will assume bash")
```